docs/control-structures.rst

```diff
@@ -51,40 +54,63 @@ write::
 
 The names of output parameters can be omitted.
 The output values can also be specified using ``return`` statements.
-The ``return`` statements are also capable of returning multiple
-values, see :ref:`multi-return`.
+
+The ``return`` statements are also capable of returning multiple values using ``return (v0, v1, ..., vn)``. The number of parameters must equal the number of output parameters.
+
 Return parameters are initialized to zero; if they are not explicitly
 set, they stay to be zero.
 
 Input parameters and output parameters can be used as expressions in
 the function body.  There, they are also usable in the left-hand side
 of assignment.
 
+.. index:: return array, return string, array, string, array of strings, dynamic array, variable sized array
+
+.. _return-array:
+
+Returning Strings and Arrays
+----------------------------
+
+You can return strings and arrays from Solidity functions, see `array_receiver_and_returner.sol <https://github.com/fivedogit/solidity-baby-steps/blob/master/contracts/60_array_receiver_and_returner.sol>`_ for an example.
```

Comment:
If we can, we should avoid linking tho this repository in the main documentation (the FAQ was a different beast). It hasn't been maintained for 2 years, so likely very outdated.

We should have a short inline example where it makes sense.

Reply:
Done, and added my own simple example